projects/storefrontapp-e2e-cypress/cypress/e2e/regression/product-carousel/product-carousel.e2e.cy.ts

```diff
@@ -99,31 +103,28 @@ context('Product carousel', () => {
             method: 'GET',
             pathname: `${Cypress.env('OCC_PREFIX')}/${Cypress.env(
               'BASE_SITE'
-            )}/products/${secureDigitalCard2gb}`,
+            )}/products/search`,
           },
-          secureDigitalCardRequestHandler
-        );
+          searchRequestHandler
+        ).as('searchProducts');
 
-        function clickNextButton(): void {
-          const cypressButton = cy.get(
-            'cx-product-carousel:first-of-type button.next'
-          );
+        visitHomepage();
 
-          cypressButton.then((button) => {
-            if (!button.is('[aria-disabled="true"]')) {
-              cypressButton.click();
-              clickNextButton();
-            }
-          });
-        }
+        cy.wait('@searchProducts'); // Verify the call was made
 
-        // Continue clicking next in the carousel until we are at the last slide.
-        clickNextButton();
```

Comment:
Note for the reviewer:
The new scrolling carousel doesn't have slides, so no need to hit forward-arrow-button (next-button) to get to the last element